control-plane/cmd/kafka-controller/main.go

```diff
@@ -76,21 +76,21 @@ func main() {
 			},
 		},
 
-		//// Namespaced broker controller
-		//injection.NamedControllerConstructor{
-		//	Name: "broker-namespaced-controller",
-		//	ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
-		//		return broker.NewNamespacedController(ctx, watcher, brokerEnv)
-		//	},
-		//},
-		//
-		//// Namespaced trigger controller
-		//injection.NamedControllerConstructor{
-		//	Name: "trigger-namespaced-controller",
-		//	ControllerConstructor: func(ctx context.Context, watcher configmap.Watcher) *controller.Impl {
-		//		return trigger.NewNamespacedController(ctx, watcher, brokerEnv)
-		//	},
-		//},
+		// Namespaced broker controller
+		injection.NamedControllerConstructor{
+			Name: "broker-namespaced-controller",
```

Comment:
I'd still like to see them called `namespaced-xxx-controller`

Reply:
Yeah, right... I forgot that. I cross my heart I will do that renaming :)